Sources/XcodeProj/Objects/Files/PBXGroup.swift

```diff
@@ -120,12 +120,26 @@ public extension PBXGroup {
     ///
     /// - Parameter name: file name.
     /// - Returns: file with the given name contained in the given parent group.
+    @available(*, deprecated, message: "Please use file(with:relativeTo:). This method assumes all names in the group are unique")
```

Comment:
Might be going a little far! But arguably the functionality is too broad, and unclear to the user

Reply:
Perhaps updating the doc comments here may suffice? It can clarify the usage and point to the other method for cases where searching by path is needed.

Reply:
Sounds good. Let me take care of that right away